app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,12 @@
+.alert-alert {
```

Comment:
Let's rename this file into something meaningful based on what elements it styles.